dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -105,14 +110,14 @@ static StaticLinkageCheckOption parseArguments(String[] arguments) throws ParseE
     ImmutableList.Builder<String> mavenCoordinates = ImmutableList.builder();
     try {
       CommandLine commandLine = parser.parse(options, arguments);
-      if ((commandLine.hasOption("b") && commandLine.hasOption("c"))
-          || (commandLine.hasOption("c") && commandLine.hasOption("j"))
+      if ((commandLine.hasOption("b") && commandLine.hasOption("a"))
+          || (commandLine.hasOption("a") && commandLine.hasOption("j"))
           || (commandLine.hasOption("j") && commandLine.hasOption("b"))) {
         throw new IllegalArgumentException(
             "One of BOM, Maven coordinates, or jar files can be specified");
       }
-      if (commandLine.hasOption("c")) {
-        String mavenCoordinatesOption = commandLine.getOptionValue("c");
+      if (commandLine.hasOption("a")) {
+        String mavenCoordinatesOption = commandLine.getOptionValue("a");
         mavenCoordinates.addAll(Arrays.asList(mavenCoordinatesOption.split(",")));
```

Comment:
Consider using [`Splitter`](https://github.com/google/guava/wiki/StringsExplained#splitter), which is safer.

Reply:
Now it uses Splitter.